package.json

```diff
@@ -27,27 +27,28 @@
   "dependencies": {
     "hoist-non-react-statics": "^1.2.0",
     "lodash.debounce": "^4.0.8",
-    "prop-types": "^15.5.8"
+    "prop-types": "^15.7.2"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-0 || ^16.0.0-0"
+    "react": "^16.0.0-0"
```

Comment:
Should we have tests for each of these versions?

Reply:
If it's possible to continue to support the older versions of React, I think that'd be nice and then yeah we'd probably want to test them. Not sure how difficult it would be to continue to support the older versions especially with renamed APIs like `UNSAFE_componentWillReceiveProps`—I also think it would be fine to drop support and call this a breaking change.

Reply:
IMO calling this a breaking change and supporting v16 is the way to go. Not sure if supporting older versions of React will have a significant benefit but as we keep improving (hooks, context providers), we won't support the older versions anyways.

Reply:
Feels weird to release a new major version that doesn't break backwards compatibility yet.

I'm fine either merging this in as is but wait to publish until we introduce actual breaking changes

or

back this change out and just release as a minor version update.

Reply:
UNSAFE_ was introduced with v16.3 I believe, so with any older version of react, it will not work. Thus my `"react": "^16.0.0-0"` is actually wrong as well and it should be a minimum v16.3. The library tests fails with older versions of react and even with v16.0. (I need to update the peer dependency version to v16.3)

If we won't be publishing it, then I do not see the benefit of merging this. It would be better to do this as a part of an improvement (such as adding hooks) and not just dependency version update but I am not sure if that would be useful. We can focus on the PR by @ghmeier and do these changes along with that and release that.

v0.14 || v15.0 will not work so we can not change this out and release a minor version

Reply:
So the other other option is to release this as a major version with these changes? I'm ok with that too. I agree with your reasoning.

Reply:
So let's do this as a major release. I will go ahead and merge the PR. Do we have documentation for the release process?

Reply:
I don't have permissions :) Feel free to merge and release if it's all good.